contracts/random/RandomConsumer.sol

```diff
@@ -65,9 +66,14 @@ contract RandomConsumer is VRFConsumerBase, Ownable, IChainlinkVRFAdmin {
      * Callback function used by VRF Coordinator
      */
     function fulfillRandomness(bytes32 requestId, uint256 randomness) internal virtual override {
+        // TODO: test interface
         IChainlinkVRFCallback(_callbackContract).fulfillRandomness(requestId, randomness);
     }
 
+    function getFee() public view returns (uint256) {

```

Comment:
What do we need getFee for? Test purposes?

Reply:
yeah just as a getter to verify contract interaction and state change. i could make the var public instead but i personally prefer this method (though i dont have a strong justification why).